xs/src/libslic3r/ConfigBase.hpp

```diff
@@ -31,20 +37,40 @@ class ConfigOptionException : public std::exception {
     public:
     const t_config_option_key opt_key;
     ConfigOptionException(const t_config_option_key _opt_key)
-        : opt_key(_opt_key) {};
+        : opt_key(_opt_key) {
+        /*
+    	void *array[10];
+    	  size_t size;
+    	    char **strings;
+    	          size_t i;
+        	  
+                size = backtrace (array, 10);
+                  strings = backtrace_symbols (array, size);
+                  
+                    printf ("Obtained %zd stack frames.\n", size);
+                    
+                      for (i = 0; i < size; i++)
+                           std::cerr<<strings[i]<<std::endl;
+                           
+                             free (strings);
+        //*/
+        };
+
+    virtual const char* what() const noexcept {
+        std::string s("Exception with the option: ");
+        s += this->opt_key;
+        return s.c_str();
+    }
+
 };
 class UnknownOptionException : public ConfigOptionException {
     using ConfigOptionException::ConfigOptionException;
+
 };
 class InvalidOptionException : public ConfigOptionException {
     using ConfigOptionException::ConfigOptionException;
     
-    public:
-    virtual const char* what() const noexcept {
-        std::string s("Invalid value for option: ");
-        s += this->opt_key;
-        return s.c_str();
-    }
+
```

Comment:
Why was this removed?

Reply:
The override of 'what' had been moved up to the base class do get a generic message of any *OptionException. Anyway, the InvalidValue override was restored in the correction push (69533af)